### DISCUSSION TITLE
unable to access on ubuntu22.04

### DISCUSSION THREAD

**Discussion by zoulee24** [rustdesk/rustdesk#8507] [General]
### Bug Description

# info
version: 1.2.6
# env
amd64 ubuntu24.04 local
# phenomenon
Unlimited window restart, multiple windows stacked, *unable to use*

### How to Reproduce

```bash
sudo dpkg -i rustdesk-1.2.6-x86_64.deb
```

### Expected Behavior

Unlimited window restart, multiple windows stacked, *unable to use*

### Operating system(s) on local side and remote side

ubuntu22.04, local

### RustDesk Version(s) on local side and remote side

1.2.6, local

### Screenshots

nope

**Comment by xlionjuan** [comment]
```
sudo apt purge rustdesk 
```

And reinstall again
